var FaerunConfig = {
  "colors": {
    colorpickerDialogProject: [
      '#3498db', '#2980b9',
      '#1abc9c', '#16a085',
      '#2ecc71', '#27ae60',
      '#f1c40f', '#f39c12',
      '#e67e22', '#d35400',
      '#e74c3c', '#c0392b',
      '#9b59b6', '#8e44ad',
      '#ecf0f1', '#bdc3c7',
      '#95a5a6', '#7f8c8d',
      '#34495e', '#2c3e50'
    ]
  },
  "server": {
    "url": 'ws://underdark.gdb.tools/underdark'
  },
  "services": {
    "fpUrls": {
      "mqn": "http://gdbtools.unibe.ch:8080/fpService/calculateFP2.jsp?smi={0}&fp=MQN",
      "apfp": "http://gdbtools.unibe.ch:8080/fpService/calculateFP2.jsp?smi={0}&fp=APfp",
      "xfp": "http://gdbtools.unibe.ch:8080/fpService/calculateFP2.jsp?smi={0}&fp=Xfp",
      "smifp": "http://gdbtools.unibe.ch:8080/fpService/calculateFP2.jsp?smi={0}&fp=SMIfp",
      "sfp": "http://gdbtools.unibe.ch:8080/fpService/calculateFP2.jsp?smi={0}&fp=Sfp",
      "ecfp4": "http://gdbtools.unibe.ch:8080/fpService/calculateFP2.jsp?smi={0}&fp=ECfp4"
    },
    "pcaUrl": "http://planes.gdb.tools/",
    "pcaParameters": {
      "mqn": {
        "database": "surechembl",
        "fingerprint": "mqn",
        "dimensions": 3,
        "binning": true,
        "resolution": 250
      },
      "apfp": {
        "database": "surechembl",
        "fingerprint": "apfp",
        "dimensions": 3,
        "binning": true,
        "resolution": 250
      },
      "smifp": {
        "database": "surechembl",
        "fingerprint": "smifp",
        "dimensions": 3,
        "binning": true,
        "resolution": 250
      }
    }
  },
  "schembl": {
    "sources": {
      "1": {
        "name": "chembl",
        "description": "A database of bioactive drug-like small molecules and bioactivities abstracted from the scientific literature.",
        "name_long": "ChEMBL",
        "base_id_url": "https://www.ebi.ac.uk/chembldb/compound/inspect/",
        "src_id": "1",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "ChEMBL",
        "src_url": "https://www.ebi.ac.uk/chembl/",
      },
      "2": {
        "name": "drugbank",
        "description": "A database that combines drug (i.e. chemical, pharmacological and pharmaceutical) data with drug target (i.e. sequence, structure, and pathway) information.",
        "name_long": "DrugBank",
        "base_id_url": "http://www.drugbank.ca/drugs/",
        "src_id": "2",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "DrugBank",
        "src_url": "http://drugbank.ca/",
      },
      "3": {
        "name": "pdb",
        "description": "The European resource for the collection, organisation and dissemination of data on biological macromolecular structures, including structures of small molecule ligands for proteins.",
        "name_long": "PDBe (Protein Data Bank Europe)",
        "base_id_url": "http://www.ebi.ac.uk/pdbe-srv/pdbechem/chemicalCompound/show/",
        "src_id": "3",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "PDBe",
        "src_url": "http://www.ebi.ac.uk/pdbe/",
      },
      "4": {
        "name": "gtopdb",
        "description": "The IUPHAR (International Union of Basic and Clinical Pharmacology)/BPS (British Pharmacological Society) Guide to PHARMACOLOGY database contains structures of small molecule ligands, peptides and antibodies, with their affinities at protein targets.",
        "name_long": "Guide to Pharmacology",
        "base_id_url": "http://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=",
        "src_id": "4",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "Guide to Pharmacology",
        "src_url": "http://www.guidetopharmacology.org",
      },
      "5": {
        "name": "pubchem_dotf",
        "description": "A subset of the PubChem DB: from the original depositor 'drugs of the future' (Prous).",
        "name_long": "PubChem ('Drugs of the Future' subset)",
        "base_id_url": "http://pubchem.ncbi.nlm.nih.gov/substance/",
        "src_id": "5",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "PubChem: Drugs of the Future ",
        "src_url": "http://pubchem.ncbi.nlm.nih.gov/sources/sources.cgi",
      },
      "6": {
        "name": "kegg_ligand",
        "description": "KEGG LIGAND is a composite DB consisting of COMPOUND, GLYCAN, REACTION, RPAIR, RCLASS, and ENZYME DBs, whose entries are identified by C, G, R, RP, RC, and EC numbers, respectively.",
        "name_long": "KEGG (Kyoto Encyclopedia of Genes and Genomes) Ligand",
        "base_id_url": "http://www.genome.jp/dbget-bin/www_bget?",
        "src_id": "6",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "KEGG Ligand",
        "src_url": "http://www.genome.jp/kegg/ligand.html",
      },
      "7": {
        "name": "chebi",
        "description": "ChEBI is a freely available dictionary of molecular entities focused on 'small' chemical compounds",
        "name_long": "ChEBI (Chemical Entities of Biological Interest).",
        "base_id_url": "http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI%3A",
        "src_id": "7",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "ChEBI",
        "src_url": "http://www.ebi.ac.uk/chebi/downloadsForward.do",
      },
      "8": {
        "name": "nih_ncc",
        "description": "Collections of plated arrays of small molecules that have a history of use in human clinical trials. Assembled by the National Institutes of Health (NIH) through the Molecular Libraries Roadmap Initiative",
        "name_long": "NIH Clinical Collection",
        "base_id_url": "null",
        "src_id": "8",
        "aux_for_url": "0",
        "base_id_url_available": "0",
        "name_label": "NIH Clinical Collection",
        "src_url": "http://nihsmr.evotec.com/evotec/",
      },
      "9": {
        "name": "zinc",
        "description": "A free database of commercially-available compounds for virtual screening, provided by the Shoichet Laboratory in the Department of Pharmaceutical Chemistry at the University of California, San Francisco (UCSF).  [Irwin and Shoichet, J. Chem. Inf. Model. 2005;45(1):177-82]",
        "name_long": "ZINC",
        "base_id_url": "http://zinc15.docking.org/substances/",
        "src_id": "9",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "ZINC",
        "src_url": "http://zinc15.docking.org",
      },
      "10": {
        "name": "emolecules",
        "description": "A free chemical structure search engine containing millions of public domain structures. Pricing, availabilities, and vendor information requires an eMolecules Plus subscription.",
        "name_long": "eMolecules",
        "base_id_url": "https://www.emolecules.com/cgi-bin/more?vid=",
        "src_id": "10",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "eMolecules",
        "src_url": "https://www.emolecules.com/",
      },
      "11": {
        "name": "ibm",
        "description": "The data are provided by IBM-NIH and include all chemistry extracted by means of text and image mining from the patent corpus (USPTO, WIPO and EPO) for patent documents published through 31-12-2010. Identifiers in UniChem are IBM compound identifiers.",
        "name_long": "IBM strategic IP insight platform and the National Institutes of Health",
        "base_id_url": "http://www-935.ibm.com/services/us/gbs/bao/siip/nih/?sid=",
        "src_id": "11",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "IBM Patent System",
        "src_url": "http://www-935.ibm.com/services/us/gbs/bao/siip/nih/",
      },
      "12": {
        "name": "atlas",
        "description": "The Gene Expression Atlas is a semantically enriched database of meta-analysis based summary statistics over a curated subset of ArrayExpress Archive, servicing queries for condition-specific gene expression patterns as well as broader exploratory searches for biologically interesting genes/samples.",
        "name_long": "Gene Expression Atlas",
        "base_id_url": "http://www.ebi.ac.uk/gxa/query?conditionQuery=",
        "src_id": "12",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "Atlas",
        "src_url": "http://www.ebi.ac.uk/gxa/home",
      },
      "14": {
        "name": "fdasrs",
        "description": "The primary goal of the FDA/USP Substance Registration System (SRS) is to unambiguously define all substances present in regulated products. Once a substance has been defined, the SRS assigns a strong identifier that is permanently associated with the substance: a UNII (Unique Ingredient Identifier). This is a a non-proprietary, free, unique, unambiguous, nonsemantic, alphanumeric identifier based on a substances molecular structure and/or descriptive information.",
        "name_long": "FDA/USP Substance Registration System (SRS)",
        "base_id_url": "http://fdasis.nlm.nih.gov/srs/ProxyServlet?mergeData=true&objectHandle=DBMaint&APPLICATION_NAME=fdasrs&actionHandle=default&nextPage=jsp/srs/ResultScreen.jsp&TXTSUPERLISTID=",
        "src_id": "14",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "FDA SRS",
        "src_url": "http://fdasis.nlm.nih.gov/srs/srs.jsp",
      },
      "15": {
        "name": "surechembl",
        "description": "SureChEMBL automatically extracts chemistry from the full text of all major patent authorities. Compounds are derived from either chemical names found in text or in chemical depictions. All SureChEMBL compounds are included, except those failing UniChem loading rules.",
        "name_long": "SureChEMBL",
        "base_id_url": "https://www.surechembl.org/chemical/",
        "src_id": "15",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "SureChEMBL",
        "src_url": "https://www.surechembl.org/search/",
      },
      "17": {
        "name": "pharmgkb",
        "description": "PharmGKB (Pharmacogenomics Knowledgebase) is a comprehensive resource that curates knowledge about the impact of genetic variation on drug response for clinicians and researchers.",
        "name_long": "PharmGKB",
        "base_id_url": "http://www.pharmgkb.org/drug/",
        "src_id": "17",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "PharmGKB",
        "src_url": "http://www.pharmgkb.org",
      },
      "18": {
        "name": "hmdb",
        "description": "The Human Metabolome Database (HMDB) is a freely available electronic database containing detailed information about small molecule metabolites found in the human body. It is intended to be used for applications in metabolomics, clinical chemistry, biomarker discovery and general education. The database is designed to contain or link three kinds of data: 1) chemical data, 2) clinical data, and 3) molecular biology/biochemistry data",
        "name_long": "Human Metabolome Database (HMDB)",
        "base_id_url": "http://www.hmdb.ca/metabolites/",
        "src_id": "18",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "Human Metabolome Database",
        "src_url": "http://www.hmdb.ca",
      },
      "20": {
        "name": "selleck",
        "description": "Selleck Chemicals is a supplier of biochemical products, including over 1,000 inhibitor products",
        "name_long": "Selleck",
        "base_id_url": "http://www.selleckchem.com/products/",
        "src_id": "20",
        "aux_for_url": "1",
        "base_id_url_available": "1",
        "name_label": "Selleck",
        "src_url": "http://www.selleckchem.com",
      },
      "21": {
        "name": "pubchem_tpharma",
        "description": "A subset of the PubChem DB: from the original depositor 'Thomson Pharma'.",
        "name_long": "PubChem ('Thomson Pharma' subset)",
        "base_id_url": "http://pubchem.ncbi.nlm.nih.gov/substance/",
        "src_id": "21",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "PubChem: Thomson Pharma ",
        "src_url": "http://www.thomson-pharma.com/",
      },
      "22": {
        "name": "pubchem",
        "description": "A database of normalized PubChem compounds (CIDs) from the PubChem Database.",
        "name_long": "PubChem Compounds",
        "base_id_url": "http://pubchem.ncbi.nlm.nih.gov/compound/",
        "src_id": "22",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "PubChem",
        "src_url": "http://pubchem.ncbi.nlm.nih.gov",
      },
      "23": {
        "name": "mcule",
        "description": "An online drug discovery platform with virtual screening and molecular modelling services.",
        "name_long": "Mcule",
        "base_id_url": "https://mcule.com/",
        "src_id": "23",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "Mcule",
        "src_url": "https://mcule.com",
      },
      "24": {
        "name": "nmrshiftdb2",
        "description": "An NMR database (web database) for organic structures and their nuclear magnetic resonance (nmr) spectra. It allows for spectrum prediction (13C, 1H and other nuclei) as well as for searching spectra, structures and other properties. Last not least, it features peer-reviewed submission of datasets by its users.",
        "name_long": "NMRShiftDB",
        "base_id_url": "http://nmrshiftdb.org/molecule/",
        "src_id": "24",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "NMRShiftDB",
        "src_url": "http://nmrshiftdb.nmr.uni-koeln.de/portal/media-type/html/user/anon/page/default.psml/js_pane/P-Home",
      },
      "25": {
        "name": "lincs",
        "description": "The LINCS DCIC facilitates and standardized the information relevant to LINCS assays as described in http://www.lincsproject.org/data/data-standards/",
        "name_long": "Library of Integrated Network-based Cellular Signatures",
        "base_id_url": "http://identifiers.org/lincs.smallmolecule/",
        "src_id": "25",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "LINCS",
        "src_url": "http://www.lincsproject.org/",
      },
      "26": {
        "name": "actor",
        "description": "ACToR (Aggregated Computational Toxicology Resource)",
        "name_long": "ACToR",
        "base_id_url": "http://actor.epa.gov/actor/chemical.xhtml?casrn=",
        "src_id": "26",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "ACToR",
        "src_url": "https://actor.epa.gov",
      },
      "27": {
        "name": "recon",
        "description": "A biochemical knowledge-base on human metabolism",
        "name_long": "Recon",
        "base_id_url": "http://vmh.uni.lu/#metabolite/",
        "src_id": "27",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "Recon",
        "src_url": "http://vmh.uni.lu",
      },
      "28": {
        "name": "molport",
        "description": "MolPort. A database designed to assist users find commercial sources of compounds. Access requires (free) registration.",
        "name_long": "MolPort",
        "base_id_url": "https://www.molport.com/shop/molecule-link/",
        "src_id": "28",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "MolPort",
        "src_url": "https://www.molport.com/shop/index",
      },
      "29": {
        "name": "nikkaji",
        "description": " Nakkaji (The Japan Chemical Substance Dictionary) is an organic compound dictionary database prepared by the Japan Science and Technology Agency (JST).",
        "name_long": "Nikkaji",
        "base_id_url": "http://jglobal.jst.go.jp/en/redirect?Nikkaji_No=",
        "src_id": "29",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "Nikkaji",
        "src_url": "http://jglobal.jst.go.jp/en/",
      },
      "31": {
        "name": "bindingdb",
        "description": "A public, web-accessible database of measured binding affinities, focusing chiefly on the interactions of proteins considered to be drug-targets with small, drug-like molecules",
        "name_long": "BindingDB",
        "base_id_url": "http://www.bindingdb.org/bind/chemsearch/marvin/MolStructure.jsp?monomerid=",
        "src_id": "31",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "BindingDB",
        "src_url": "https://www.bindingdb.org/bind/index.jsp",
      },
      "32": {
        "name": "comptox",
        "description": "The foundation of chemical safety testing relies on chemistry information such as high-quality chemical structures and physicochemical properties. This information is used by scientists to predict the potential health risks of chemicals.The CompTox Dashboard is part of a suite of dashboards developed by EPA to help evaluate the safety of chemicals. It provides access to a variety of data and information on over 700,000 chemicals currently in use and of interest to environmental researchers. Within the CompTox Dashboard, users can access chemical structures, experimental and predicted physicochemical and toxicity data, and additional links to relevant websites and applications. It maps curated physicochemical property data associated with chemical substances to their corresponding chemical structures",
        "name_long": "EPA (Environmental Protection Agency) CompTox Dashboard",
        "base_id_url": "https://comptox.epa.gov/dashboard/",
        "src_id": "32",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "EPA CompTox Dashboard",
        "src_url": "https://comptox.epa.gov/dashboard/",
      },
      "33": {
        "name": "lipidmaps",
        "description": "LIPID Metabolites And Pathways Strategy (LIPID MAPS) is a multi-institutional effort created to identify and quantitate, using a systems biology approach and sophisticated mass spectrometers, all of the major, and many minor, lipid species in mammalian cells, as well as to quantitate the changes in these species in response to perturbation",
        "name_long": "LipidMaps",
        "base_id_url": "http://www.lipidmaps.org/data/LMSDRecord.php?LMID=",
        "src_id": "33",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "LipidMaps",
        "src_url": "http://www.lipidmaps.org",
      },
      "34": {
        "name": "drugcentral",
        "description": "DrugCentral is an online drug information resource created and maintained by Division of Translational Informatics at University of New Mexico, providing information on active ingredients chemical entities, pharmaceutical products, drug mode of action, indications, pharmacologic action",
        "name_long": "DrugCentral",
        "base_id_url": "http://drugcentral.org/drugcard/",
        "src_id": "34",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "DrugCentral",
        "src_url": "http://drugcentral.org",
      },
      "35": {
        "name": "carotenoiddb",
        "description": "A Database of information on naturally occurring carotenoids from many organisms, extracted  from the literature.",
        "name_long": "Carotenoid Database",
        "base_id_url": "http://carotenoiddb.jp/Entries/",
        "src_id": "35",
        "aux_for_url": "0",
        "base_id_url_available": "1",
        "name_label": "CarotenoidDB",
        "src_url": "http://carotenoiddb.jp/index.html",
      }
    }
  }
}
